src/InteractiveWindow/Editor/InteractiveWindow_UIThread.cs

```diff
@@ -778,7 +772,7 @@ public void ExecuteInput()
                         completionSession.Commit();
                     }
 
-                    SubmitAsync();
+                    var dummy = SubmitAsync();
```

Comment:
It eliminates a warning about dropping the result of an async method.  I tried to convey that with the name, but I can add a comment if you prefer.